src/cli/utils/sharepoint.go

```diff
@@ -105,6 +108,20 @@ func ValidateSharePointRestoreFlags(backupID string, opts SharePointOpts) error
 		}
 	}
 
+	if _, ok := opts.Populated[flags.ListTemplateFN]; ok {
+		found := false
+
+		for _, wu := range opts.ListTemplate {
+			if wu == flags.InvalidListTemplate {
+				found = true
+			}
+		}
+
+		if !found {
+			return clues.New(fmt.Sprintf("%s list template not present", flags.InvalidListTemplate))
```

Comment:
per code standards, errors should not include format statements.

aside from that, I'm not sure what the purpose of this error is: to require the flag parameters to contain a certain value?  That's a poor UX for end users, since it's not normal to require values in a flag.  I would be confused if I told the CLI to use a certain (valid) flag value and was given an error in response due to my command missing some _other_ valid flag value.  It's too much opportunity for the user to give themselves a bad experience.

Reply:
Instead of returning an error for not finding the 'InvalidListTemplate', maybe we can add it to the opts.ListTemplate.